### ISSUE TITLE
Dynamic Api Version per endpoint possible?

### ISSUE THREAD

**Issue by abelfiore** [proudmonkey/AutoWrapper#84]
Would it be possible to enhance AutoWrapper to dynamically show the Api Version of each endpoint (they can all theoretically be different)? I'm assuming the ApiVersion option is currently a static global version for the entire api.

**Comment by proudmonkey** [comment]
The `ApiResponse` object has an overload that allows you to pass in the API version. Have you tried that?

**Comment by abelfiore** [comment]
Yes, I saw that, but was wondering if AutoWrapper could support a way to do this without explicitly doing anything in the controller endpoints. In other words, detect the api version of the method executing.
